R/step_epi_shift.R

```diff
@@ -30,8 +29,14 @@
 #' @param id A unique identifier for the step
 #' @template step-return
 #'
-#' @details The step assumes that the data are already _in the proper sequential
-#'  order_ for shifting.
+#' @details The step assumes that the data's `time_value` column is already _in
+#'   the proper sequential order_ for shifting.
+#'
```

Comment:
Is this true? I though it just added to the time_value and joined. In that case,
it wouldn't matter what order it was in.

Reply:
apparently not. This was a comment from quite a long time ago that just got moved around. I'll delete it, it must've been held over from a time when that was true.